microsoft/testsuites/network/sriov.py

```diff
@@ -693,6 +693,8 @@ def verify_sriov_interrupts_change(self, environment: Environment) -> None:
         server_iperf3.run_as_server_async()
         client_interrupt_inspector = client_node.tools[InterruptInspector]
         for _, client_nic_info in vm_nics[client_node.name].items():
+            if client_nic_info.has_pci_device and not client_nic_info.has_pci_module:
```

Comment:
should it check pci_module enough?

Reply:
I suppose so

Reply:
if so, will you change it?

Reply:
now I use pci_slot + lower to judge whether continue following testing

Reply:
we defined `is_pci` when designing. Is it should be used here? Either use `is_pci` to track the upper, or track both upper and lower. The pci_slot is not a good part to track for this purpose. It's not clear enough. The `module_name` doesn't specify if it's a PCI device. So we need a variable or property to represent the information.